content/en/docs/aws/customizing-aws.md

```diff
@@ -87,4 +87,4 @@ Please see [this section](/docs/aws/iam-for-sa)
 Please see [this section](/docs/aws/private-access)
 
 ### Customize Logging
-Please see [this section](/docs/aws/logging)
\ No newline at end of file
+Please see [this section](/docs/aws/logging)
```

Comment:
```suggestion

Go to the [Add logging support for Kubeflow](/docs/aws/logging) guide for more information.
```

("see" -> "refer to" or "go to" for accessibility)
(use full name of guides for better UX)